raiden-cli/src/index.ts

```diff
@@ -368,6 +373,19 @@ async function checkDisclaimer(accepted?: boolean): Promise<void> {
   assert(accepted, 'Disclaimer not accepted!');
 }
 
+async function getUdcAddressIfUnstable(
+  ethRpcEndpoint: string,
+  developmentEnvironment: string,
+): Promise<string | undefined> {
+  const provider = new JsonRpcProvider(ethRpcEndpoint);
```

Comment:
I´d prefer to not have yet another provider instantiated here. Two possible alternatives: either assume unstable implies goerli, and let it crash if the user asked for unstable but isn´t connected to goerli, since this would be an user error; or we could finally mind that `--network-id` param we´ve been neglecting so far and validate it from there.